### PR TITLE
ShowCategoryIconInNotification

### PR DESCRIPTION
**Pull request by jeppekikt**

### CODE REVIEW

File: TaskSharper.Shared/Logging/LogConfiguration.cs [hunk 1]
```diff
@@ -42,10 +42,10 @@ private static LoggerConfiguration BaseConfig()
...
                 .MinimumLevel.Information();
 
-            if (ConnectionHelper.CheckConnectionElasticsearch(_elasticSearchUrl))
-            {
-                logger.AddElasticsearch(_elasticSearchUrl);
-            }
+            //if (ConnectionHelper.CheckConnectionElasticsearch(_elasticSearchUrl))
+            //{
+            //    logger.AddElasticsearch(_elasticSearchUrl);
+            //}
```
Comment (madsmarcussen) [inline TaskSharper.Shared/Logging/LogConfiguration.cs]: Revert this...

**Review by madsmarcussen — CHANGES_REQUESTED** [review]
Revert the out-commented code and it's good! 🥇 👍

**Review by madsmarcussen — APPROVED** [review]
Approved.

File: TaskSharper.Shared/Logging/LogConfiguration.cs (hunk 1 above)
Comment (jeppekikt) [reply TaskSharper.Shared/Logging/LogConfiguration.cs]: Done

**Pull request merged**